data-access-layer/eventmesh/ApiServerClient.js

```diff
@@ -504,7 +591,7 @@ class ApiServerClient {
     };
     return Promise.try(() => this.init())
       .then(() => apiserver.api[CONST.APISERVER.CONFIG_MAP.API_VERSION]
-        .namespaces(CONST.APISERVER.NAMESPACE)[CONST.APISERVER.CONFIG_MAP.RESOURCE_TYPE].post({
+        .namespaces(CONST.APISERVER.DEFAULT_NAMESPACE)[CONST.APISERVER.CONFIG_MAP.RESOURCE_TYPE].post({
```

Comment:
why is config map always in default name space? why not option here to create within input namespace?

Reply:
This is not part of this PR. Only the constant name was changed. Hence let's keep it out of this PR. But I agree with you. :)